### PR TITLE
DL-13731 : remove the govt-gateway references

### PR DESCRIPTION
**Pull request by kannanEkambaram**
!!! :guardsman: PLEASE UPDATE THE TEXT BELOW AND REMOVE THIS LINE :guardsman: !!!

# Descriptive Title - including Jira story or Github issue number

**Bug fix / New feature** (delete as appropriate)

Please include a summary / description of the change and which issue it fixes.  Include any relevant user needs and context.

## Checklist

* [ ] I've included appropriate tests with any code I've added
* [ ] I've executed the acceptance test pack locally to ensure there are no regressions
* [ ] I've added my code using logical, atomic commits, squashing as appropriate - including the Jira issue number in the commit message
* [ ] I've run a dependency check to ensure all dependencies are up to date

### CODE REVIEW

File: conf/application.conf [hunk 1]
```diff
@@ -81,7 +81,7 @@ microservice {
       }
 
       gg-reg-fe {
-        url = "http://localhost:8571"
+        url = "http://localhost:9553/bas-gateway/register"
```
Comment (chaithupejjai) [inline conf/application.conf:84]: is this port and url changed recently? how about other environments?

File: app/controllers/EligibleController.scala [hunk 2]
```diff
@@ -35,12 +35,11 @@ class EligibleController @Inject()(controllerComponents: MessagesControllerCompo
...
     val companyRegURI = s"${appConfig.compRegFEURL}${appConfig.compRegFEURI}"
     val crfePostSignIn = s"$companyRegURI${appConfig.postSignInUrl}"
-    val ggrf = "government-gateway-registration-frontend"
     val accountType = "accountType=organisation"
     val origin = "origin=company-registration-frontend"
-    val continueURL = s"continue=${URLEncoder.encode(s"$crfePostSignIn", "UTF-8")}"
+    val continueURL = s"continueUrl=${URLEncoder.encode(s"$crfePostSignIn", "UTF-8")}"
 
-    s"${appConfig.ggMakeAccountUrl}/$ggrf?$accountType&$continueURL&$origin"
+    s"${appConfig.ggMakeAccountUrl}?$accountType&$continueURL&$origin"
```
Comment (chaithupejjai) [inline app/controllers/EligibleController.scala:42]: How this ggrf to accountType change is applicable? is this path param used somewehere in these files?

File: conf/application.conf (hunk 1 above)
Comment (kannanEkambaram) [reply conf/application.conf:84]: The field gg-reg-fe is used only for local evn testing. Its been replaced with "" (null values) in the app-config-base. So, it won't impact the lower environments.

File: app/controllers/EligibleController.scala (hunk 2 above)
Comment (kannanEkambaram) [reply app/controllers/EligibleController.scala:42]: Since its replaced with "" in the app-config-base, it won't impact anything.

**Pull request merged**